protokt-codegen/src/main/kotlin/com/toasttab/protokt/codegen/model/FieldType.kt

```diff
@@ -95,43 +105,57 @@ private sealed class TypeImpl {
 
 private sealed class Nonscalar(
     override val kotlinRepresentation: KClass<*>? = null,
-    override val ktRepresentation: KClass<*>? = null
+    override val ktRepresentation: KClass<*>? = null,
+    override val isGeneratedType: Boolean = false,
+    override val defaultValue: CodeBlock = CodeBlock.of("null")
```

Comment:
I think it ended up nice to not bind these models to a particular code generator implementation - can this be implemented with a `when()` statement in the annotator(s) that needs it?